### ISSUE TITLE
`docker service create -i`

### ISSUE THREAD

**Issue by atedja** [docker/docker#32300]
I have been searching online for anyone to have mentioned this ever so slightly, but I can't seem to find the `-i` option when running `docker service create`.  I am not sure if this is a deprecated feature, but you can do the following `docker run -it`, and when running `docker inspect`, you can see that the container `Tty` and `OpenStdin` are set to `true`.  `docker service create` [documentation](https://docs.docker.com/engine/reference/commandline/service_create/) only has `-t`, but  missing its `-i`. Nothing saying this feature is being deprecated, or if maybe it's simply forgotten.

In other words, how should one enable `OpenStdin` from `docker service`

**Comment by thaJeztah** [comment]
Correct, "interactive" services are (currently) not supported. There are a couple of reasons for this;

- the `service create` command is asynchronous, so when the command completes, this only means the instructions are "received". In the background, docker looks to available nodes to place tasks on, pulls images on those nodes, and deploys the tasks there (work is done on having a "synchronous" option https://github.com/docker/docker/pull/31144)
- a service can be (and usually is) backed by _multiple_ tasks (containers); which container should `stdin` be attached to?
- contrary to a "regular" `docker run`, the daemon does not directly interact with the nodes that the containers run on (also see earlier bullet on "asynchronous"); the "agents" are responsible for actually deploying the containers, without blocking the manager that you connected to (and deployed the service from).

We have an "epic" issue tracking options that are not (or not _yet_) supported on services; https://github.com/docker/docker/issues/25303, and I just added your issue to that list
